packages/mui/src/components/ElementAvatar/ElementAvatar.test.tsx

```diff
@@ -57,15 +57,15 @@ describe('<ElementAvatar>', () => {
       <ElementAvatar
         userId="@user-alice"
         displayName="Alice"
-        avatarUrl={'mxc://alice'}
+        avatarUrl={'mxc://example.com/imageAlice'}
       />,
     );
 
     await expect(
       screen.findByRole('presentation', { hidden: true }),
     ).resolves.toHaveAttribute(
       'src',
-      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/alice/i),
+      expect.stringMatching(/\/_matrix\/media\/r0\/thumbnail\/.*\/imageAlice/i),
```

Comment:
This should also contain the server name https://spec.matrix.org/v1.12/client-server-api/#get_matrixclientv1mediathumbnailservernamemediaid